### DISCUSSION TITLE
Turn off sounds

### DISCUSSION THREAD

**Discussion by shift838** [ZuluSCSI/ZuluSCSI-firmware#415] [Q&A] [answered]
Is there a way in the .ini file to turn off the speaker?

**Comment by aperezbios** [comment, marked as answer]
@shift838 to disable the buzzer sounds, you have two options. First, you can disable the piezo buzzer by removing the two-pin jumper that feeds the input to the buzzer, which is a hardware-only (non-ini-file) method. This can also be wired up to an externally accessible switch, if you want an easy way to be able to enable and disable it at will.

Via the INI file, your only option is to disable the LED activity entirely, which would also result in the disabling of the buzzer, because both are driven by the same pin from the RP2040 microcontroller.

In your zuluscsi.ini file, you would set:

```DisableStatusLED=1```

**Reply by shift838** [reply]
if the speaker had more of the typical old school hard disk sounds emulated then it would be tolerable, but the load beep when it is doing any file operations gets on your nevers after a while, especially if you are copying a lot of files.

**Reply by aperezbios** [reply]
@shift838 if you're getting a "loud beep" then you're using a piezoelectric buzzer that has its own tone generator, so the noise itself is not being generated by the ZuluSCSI. Without knowing the exact make and model of the buzzer you're using I can't comment on the specifics, though.

**Comment by shift838** [comment]
I’m using whatever came in it.  But that sounds accurate.  Which speaker do
you recommend?

**Reply by aperezbios** [reply]
Where did you purchase the board from? It sounds like a third party installed it, not Rabbit Hole Computing.

**Comment by shift838** [comment]
I would have to check,  I thought it was RabbitHole.  But it’s been a
while.  I can replace the speaker just need to know which one to get.
